src/js/App/Header/AppFilter/useAppFilter.js

```diff
@@ -3,7 +3,6 @@ import { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
 import { isBeta } from '../../../utils';
 import { evaluateVisibility } from '../../../utils/isNavItemVisible';
-import { navigationFileMapper } from '../../../utils/useNavigation';
 
 export const requiredBundles = ['application-services', 'openshift', 'insights', 'edge', 'ansible', 'settings'];
 const bundlesOrder = ['application-services', 'openshift', 'rhel', 'edge', 'ansible', 'settings', 'cost-management', 'subscriptions'];
```

Comment:
Just FYI if you are not going to add the `cloud-tutorials` in here it won't be shown in the app filter.

Reply:
That's okay! We'd like to keep it hidden for now.